[2022-12-03 14:55:14,875] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:30:00+00:00 [queued]>
[2022-12-03 14:55:15,921] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:30:00+00:00 [queued]>
[2022-12-03 14:55:16,004] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:55:16,017] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-03 14:55:16,018] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:55:17,444] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 18:30:00+00:00
[2022-12-03 14:55:17,670] {standard_task_runner.py:52} INFO - Started process 662 to run task
[2022-12-03 14:55:17,955] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T18:30:00+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpa9c0vcsd', '--error-file', '/tmp/tmpczcovo1f']
[2022-12-03 14:55:18,098] {standard_task_runner.py:80} INFO - Job 158: Subtask get_last_id
[2022-12-03 14:55:21,897] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:30:00+00:00 [running]> on host b41792e8e843
[2022-12-03 14:55:29,668] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T18:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T18:30:00+00:00
[2022-12-03 14:55:30,761] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-03 14:55:32,091] {python.py:173} INFO - Done. Returned value was: None
[2022-12-03 14:55:35,148] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T183000, start_date=20221203T145514, end_date=20221203T145535
[2022-12-03 14:55:38,035] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-03 14:55:40,620] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
